plugins/module_utils/network/eos/config/bgp_global/bgp_global.py

```diff
@@ -234,6 +239,20 @@ def _compare_vrfs(self, want, have):
                     )
                     self.commands.append("exit")
 
+    def _check_af (self, vrf):
+        af_present = False
+        if self._connection:
+            config_lines = Bgp_globalFacts.get_config(self, self._connection).splitlines()
```

Comment:
Is there any reason why we're doing this instead of just self._connection.get_config()?

Reply:
yes, for the unit tests of all modules, I mock the `get_config` method from the respective facts class _( in bgp_global module's case, it from Bgp_globalFacts class)_ . I have to check if mocking the `get_config` method from eos.py works for unit tests.

Reply:
We can also add a get_config() method in config class rather than importing the facts class only for this method?